docs/proposals/rescheduling-for-critical-pods.md

```diff
@@ -0,0 +1,116 @@
+<!-- BEGIN MUNGE: UNVERSIONED_WARNING -->
+
+<!-- BEGIN STRIP_FOR_RELEASE -->
+
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+
+<h2>PLEASE NOTE: This document applies to the HEAD of the source tree</h2>
+
+If you are using a released version of Kubernetes, you should
+refer to the docs that go with that version.
+
+Documentation for other releases can be found at
+[releases.k8s.io](http://releases.k8s.io).
+</strong>
+--
+
+<!-- END STRIP_FOR_RELEASE -->
+
+<!-- END MUNGE: UNVERSIONED_WARNING -->
+
+# Rescheduler: guaranteed scheduling of critical addons
+
+## Motivation
+
+In addition to Kubernetes core components like api-server, scheduler, controller-manager running on a master machine
+there is a bunch of addons which due to various reasons have to run on a regular cluster node, not the master.
```

Comment:
No, my point was to clarify why some run on master and some can't

On Jul 24, 2016 12:17 PM, "David Oppenheimer" <notifications@github.com>
wrote:

> In docs/proposals/rescheduling-for-critical-pods.md
> <https://github.com/kubernetes/kubernetes/pull/29195#discussion_r71993699>
> :
>
> > +
> > +Documentation for other releases can be found at
> > +[releases.k8s.io](http://releases.k8s.io).
> > +</strong>
> > +--
> > +
> > +<!-- END STRIP_FOR_RELEASE -->
> > +
> > +<!-- END MUNGE: UNVERSIONED_WARNING -->
> > +
> > +# Rescheduler: guaranteed scheduling of critical addons
> > +
> > +## Motivation
> > +
> > +In addition to Kubernetes core components like api-server, scheduler, controller-manager running on a master machine
> > +there is a bunch of addons which due to various reasons have to run on a regular cluster node, not the master.
>
> @thockin <https://github.com/thockin> I agree there's a fundamental
> difference, but do you think it affects whether they are critical to keep
> scheduled?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/29195/files/a135c597401e8d9e12e19845600b8061ffdb4b7c#r71993699>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AFVgVNQ7Lniq9ALktA7Rr5zQyU9uXbhCks5qY7pDgaJpZM4JPqoM>
> .
>